sql/log.cc

```diff
@@ -709,19 +709,23 @@ bool File_query_log::open()
     goto err;
 
   {
+    char log_creation_time[iso8601_size];
+    make_iso8601_timestamp(log_creation_time);
+
     char *end;
-    size_t len=my_snprintf(buff, sizeof(buff), "%s, Version: %s (%s). "
+    size_t len=my_snprintf(buff, sizeof(buff), "%s, Version: %s (%s), Time: %s. "
 #ifdef EMBEDDED_LIBRARY
                         "embedded library\n",
-                        my_progname, server_version, MYSQL_COMPILATION_COMMENT
+                        my_progname, server_version, MYSQL_COMPILATION_COMMENT,
+                        log_creation_time
 #elif _WIN32
                         "started with:\nTCP Port: %d, Named Pipe: %s\n",
                         my_progname, server_version, MYSQL_COMPILATION_COMMENT,
-                        mysqld_port, mysqld_unix_port
+                        log_creation_time, mysqld_port, mysqld_unix_port
```

Comment:
will this cause any parsing issues?

Reply:
At least pt-query-digest works fine with this. It's difficult to make any assumptions as for all other possible parsers.

Reply:
is there a version on this file somewhere? Any on-disk file should have a version number on the header. Rule 1 on designing disk formats.

Reply:
I don't see any log header format version for slow log. There is only server version in the header

`percona-server/5.7/cmake-build-debug/sql/mysqld, Version: 5.7.33-36-debug-log (Source distribution), Time: 2021-07-05T19:55:56.317272Z. started with: Tcp port: 13001 Unix socket: percona-server/5.7/cmake-build-debug/mysql-test/var/tmp/mysqld.1.sock`

Reply:
ok, may the server version could be used by parsers (if any) to detect the change.